src/20250430_Multi_LRTAndWstETHUnification/LRTAndWstETHUnification.md

```diff
@@ -0,0 +1,375 @@
+
+## Motivation
+
+This publication incorporates the feedback and recent discussions into a single holistic publication. For ease of reference, those publications are referenced below:
+
+- [[ARFC] wstETH and weETH E-Modes and LT/LTV Adjustments on Ethereum, Arbitrum, Base - 03.12.25](https://governance.aave.com/t/arfc-wsteth-and-weeth-e-modes-and-lt-ltv-adjustments-on-ethereum-arbitrum-base-03-12-25/21370)
+- [[ARFC] rsETH LTV & LT Update](https://governance.aave.com/t/arfc-rseth-ltv-lt-update/21305)
+- [[ARFC] Core Instance - Add ezETH and update rsETH eMode Parameters](https://governance.aave.com/t/arfc-core-instance-add-ezeth-and-update-rseth-emode-parameters/21505)
+- [[Risk Stewards]wstETH/wETH eMode Update - Ethereum, Arbitrum & Base Instances](https://governance.aave.com/t/risk-stewards-wsteth-weth-emode-update-ethereum-arbitrum-base-instances/21333)
+
+Each of the following sub-sections presents insights into how each parameter is to be adjusted.
+
+### Prime Instance - wstETH and WETH eMode
+
+The Prime instance, previously Lido instance, presents a tailored wstETH eMode configuration that offers enhanced capital efficiency relative to other instances of Aave v3. With a Liquidation Threshold (LT) of 96.50% for wstETH on Prime relative to 95.00% elsewhere, a position with a Health Factor of 1.01 can support a leverage ratio of 22.44 on Prime relative to 16.83 on other instances of Aave Protocol.
+
+With all other variables held constant, a small difference in the wstETH deposit yield on Prime relative to the Core instance, has a meaningful impact on the overall return of the wstETH/WETH yield strategy. With a strong focus on sustaining a wstETH deposit yield derived from LRT/wstETH leverage, the wstETH/WETH yield strategy is expected to outperform on Prime relative to other venues supporting the same wstETH/WETH strategy.
+
+Provided suitable ETH liquidity is available, implementing favourable eMode terms on Prime relative to other instances of Aave is expected to lead to significant growth in wstETH deposits. A LT of 96.50% exceeds the previous forum discussion supportive of implementing a 96.00% LT, whilst also isolating the LT increase specifically to the Prime instance that reflects a clear focus to progress Prime into a sustainable ETH correlated instance of Aave v3.
+
+Reference: [[Risk Stewards]wstETH/wETH eMode Update - Ethereum, Arbitrum & Base Instances](https://governance.aave.com/t/risk-stewards-wsteth-weth-emode-update-ethereum-arbitrum-base-instances/21333)
+
+#### wstETH/wETH Legacy eMode Update
+
+|       Parameter       |    Value     |
+| :-------------------: | :----------: |
+|         Asset         | wstETH, WETH |
+|        Max LTV        |     93.5     |
+| Liquidation Threshold |     95.5     |
+|  Liquidation Penalty  |    1.00%     |
+
+#### New wstETH/wETH v3.2 liquid eMode
```

Comment:
Why is this needed. Looks really messy, imo better to check with risk teams if you cannot just alter the existing one and not add a new one. Will check with chaos.